bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/internal/item/ItemResource.java

```diff
@@ -176,6 +178,15 @@ protected void unsetDTOMapper(DTOMapper dtoMapper) {
         this.dtoMapper = dtoMapper;
     }
 
+    @Reference(cardinality=ReferenceCardinality.MANDATORY, policy=ReferencePolicy.STATIC)
+    protected void setLocaleService(LocaleService localeService) {
+        this.localeService = localeService;
+    }
+    
+    protected void unsetLocaleService(LocaleService localeService) {
+        this.localeService = null;
```

Comment:
with this you should have an error in your IDE: `Null type mismatch: required '@NonNull LocaleService' but the provided value is null`